### PR TITLE
stable/airflow: revert initdb order change

### PR DESCRIPTION
**Pull request by memory**
The init order change in https://github.com/helm/charts/pull/21368
was the wrong solution to the problem at hand (circular dependencies
between a custom db hook and initdb) and can create init failures
when installing with a blank db.  Reverting.

Signed-off-by: Nathan J. Mehl <n@oden.io>

#### What this PR does / why we need it:

https://github.com/helm/charts/pull/21368 created init failures with a blank db. :(

#### Which issue this PR fixes
n/a

#### Special notes for your reviewer:
@gsemet I'm still iterating on a larger fix for these issues (https://github.com/helm/charts/pull/21425) but the current initdb behavior is definitely broken (and my apologies for that!) so this should probably go in before 21425.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by gsemet** [comment]
Can you rebase ? and update the minor version please

**Comment by memory** [comment]
@gsemet rebased and updated

File: stable/airflow/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 6.4.0
+version: 6.4.1
```
Comment (gsemet) [inline stable/airflow/Chart.yaml]: 6.5.0

Comment (memory) [reply stable/airflow/Chart.yaml]: okay, sure!

Comment (memory) [reply stable/airflow/Chart.yaml]: Done.

**Pull request merged**